.gitignore

```diff
@@ -1,3 +1,4 @@
-# Custom ignore rules:
-## Python general
```

Comment:
I prefer following style: https://github.com/edx/edx-ora2/blob/master/.gitignore
But if you really insist I'll add header comment.